infoblox_client/feature.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2015 Infoblox Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+import six
+
+from infoblox_client import exceptions as ib_ex
+
+FEATURE_VERSIONS = [
+    {'feature': 'create_ea_def', 'min_ver': '2.2'},
```

Comment:
I considered that too. That would actually be cleaner.

But, I wanted to allow for future enhancement to include 'max_ver' (that's why I called it 'min_ver' instead of just 'version'/'req_version') if/when it becomes necessary.

I'm ok either way. Please let me know what you think.